.evergreen.yml

```diff
@@ -1,3 +1,4 @@
+exec_timeout_secs: 7200 # automatically fail any task if it takes longer than 120 minutes to finish.
```

Comment:
I saw our times on evergreen and I think I'd be happier if we lower this to an hour or less, if it's taking longer there's definitely something wrong with atlas and not us